cpp_api/eclipse.sh

```diff
@@ -6,4 +6,4 @@ if [ $# -gt 0 ] ; then
     exit 1
 fi
 
-cmake -G"Eclipse CDT4 - Unix Makefiles" -D CMAKE_BUILD_TYPE=Debug `pwd`
+# cmake -G"Eclipse CDT4 - Unix Makefiles" -D CMAKE_BUILD_TYPE=Debug `pwd`
```

Comment:
I know this was changed due to some `Makefile` related issues but I don't believe it is still required to achieve the desired effect. It would be ideal to revert this unless it is still needed to achieve the desired effect of disabling the build for the `cpp_api`.